addons/playersuppression/functions/fnc_modulePlayerSuppression.sqf

```diff
@@ -0,0 +1,38 @@
+#include "script_component.hpp"
+
+private ["_logic"];
+_logic = _this select 0;
+GVAR(intensity)         = _logic getVariable "intensity";
+GVAR(enablePlayerSuppression) = true;
+if (isDedicated) exitWith {};
+
+GVAR(threshold)         = 0;	 // Changing value
+GVAR(maxValue)          = 30;	 // max. value that can be reached
+GVAR(border)            = 8;	 // The border value for the effects to start
```

Comment:
If these are constants, they should be #define's in script_component.hpp instead so they are preprocessed in instead of a memory deref.